pegasus/sites.v3/code.org/views/theme_common_head_before.haml

```diff
@@ -10,6 +10,8 @@
 %link{rel:'apple-touch-icon', href:'/images/apple-touch-icon-precomposed.png'}
 
 %script{src:'https://www.googleoptimize.com/optimize.js?id=OPT-KBX3C3L'}
+-# Import FONT AWESOME PRO for all pegasus/sitev3/code.org pages
+%script{src: "https://kit.fontawesome.com/ea2d9d4413.js", crossorigin: "anonymous", async: true}
```

Comment:
We should also make sure we aren't using a third-party domain for fonts on all pegasus sites as well.  The reason is that we want to make sure our site works in locations that have very strict firewalls, where they might have only permitted our domain but not other ones.

Reply:
issue solved)